### DISCUSSION TITLE
Copilot 1.1.27.1708 (Jetbrains webstorm 2022.1.2) sign in problem even eligible to use GitHub Copilot for free

### DISCUSSION THREAD

**Discussion by Misaka-0x447f** [community/community#19494] [Copilot Conversations]
![image](https://user-images.githubusercontent.com/15797507/175821192-f20620c0-1499-495b-9682-d5a8bd9db92f.png)
![image](https://user-images.githubusercontent.com/15797507/175821204-6300aab8-7a8e-490e-b198-a9e70d9fc3db.png)

Plugin version 1.1.27.1708 (Jetbrains webstorm)

Visit https://github.com/github-copilot/tp_signup then select "Continue without payment details" does not work for me.

This problem has been last since the payment plan come out.

## Webstorm diagnostic

WebStorm 2022.1.2
Build #WS-221.5787.30, built on June 1, 2022
Licensed to Kumomi Misaka
Subscription is active until January 16, 2023.
Runtime version: 11.0.15+10-b2043.56 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 5.13.0-51-generic
GC: G1 Young Generation, G1 Old Generation
Memory: 1928M
Cores: 16
Registry:
    ide.balloon.shadow.size=0
    ide.images.show.chessboard=true

Non-Bundled Plugins:
    org.sylfra.idea.plugins.linessorter (1.0.1)
    org.misaka.plugins.webstorm-keymap (2.2.0)
    org.jetbrains.intellij.scripting-ruby (1.0)
    dev.meanmail.plugin.nginx-intellij-plugin (2021.3-2022.1)
    commit-template-idea-plugin (1.2.0)
    com.shetline.ligatures-limited (1.0.4)
    com.github.copilot (1.1.27.1708)
    zielu.gittoolbox (212.9.0)
    mobi.hsz.idea.gitignore (4.4.0)
    mdx.js (1.0.221)
    jones.restarteslintaction.restart-eslint-action (0.0.1)
    org.i18nally.jetbrainsideplugin (1.4.1.2)
    com.ppolivka.plugin.gitcleaner (1.1.4)

Current Desktop: KDE

**Comment by melvincwng** [comment]
@Misaka-0x447f, seems like you are facing the same issue in #19213.
 
If that's the case, can you try the following:

1. Log out of your GitHub account in WebStorm IDE
2. Close your WebStorm IDE editor
3. Relog in back to your Github account using WebStorm IDE
4. This should regenerate the auth token from Github's servers
5. If you have access to Github Copilot, it should be working now.

Give it a try and hopefully, they work for you too. You can click on that discussion thread #19213 for more information as well.

As a last resort, raise a ticket to [Github Support](https://support.github.com/contact/account) and provide them several working days to investigate and get back to you.

Hope this helps resolve your issue.

**Reply by melvincwng** [reply]
@Misaka-0x447f, if this has helped you, please 'Mark as answer' so the rest of the community can be aware of the solution should they face similar issues in the future.

Thanks, and have a great day ahead!

**Reply by Misaka-0x447f** [reply]
Unfortunately I cannot log out my account. I'm not logged in.
![image](https://user-images.githubusercontent.com/15797507/175822026-5340b45f-3943-4d37-99ee-d1abbd52a12d.png)

**Reply by melvincwng** [reply]
@Misaka-0x447f, that's the reason why you couldn't even use GitHub Copilot to begin with.

You have to login in order to use Github Copilot.

Please login, and test if Github Copilot is working on your end. Follow this step-by-step guide [here](https://docs.github.com/en/copilot/getting-started-with-github-copilot/getting-started-with-github-copilot-in-a-jetbrains-ide) on how to set up Github Copilot on Jetbrain IDEs.

**Reply by Misaka-0x447f** [reply]
> @Misaka-0x447f, that's the reason why you couldn't even use GitHub Copilot to begin with.
> 
> You have to login in order to use Github Copilot.
> 
> Please login, and test if Github Copilot is working on your end. Follow this step-by-step guide [here](https://docs.github.com/en/copilot/getting-started-with-github-copilot/getting-started-with-github-copilot-in-a-jetbrains-ide) on how to set up Github Copilot on Jetbrain IDEs.

No, I logged in, then that error appears, and then the plugin return to not logged in status.

**Reply by melvincwng** [reply]
@Misaka-0x447f, sounds like a connectivity issue on your end.

Are you able to restart your code editor, and try to log back in? And then follow the steps which I've just mentioned.

**Reply by melvincwng** [reply]
@Misaka-0x447f, as a last resort, if all fails:

- Kindly raise a ticket to [Github Support](https://support.github.com/contact/account) and provide them several working days to investigate and get back to you.

**Reply by Misaka-0x447f** [reply]
Okay I was pretend this is yours support forum. Thanks for redirect.
